### ISSUE TITLE
Scoop branding is unnecessary

### ISSUE THREAD

**Issue by God-damnit-all** [71/scoop-better-shimexe#7]
I found this project by looking for something like what Chocolatey does with its shims without having to use Chocolatey's PowerShell modules. I don't use Scoop at all. This works much more reliably than alternatives like doskey aliases or batch files.

Furthermore it'd be good to describe how to create a .shim file for those who have never used Scoop before. (Honestly this would be good to include either way, but especially if you would be interested in removing the Scoop association.)

**Comment by 71** [comment]
Makes sense. However it'll require me to rewrite (at least partly) the README and to add some instructions, so... TODO.

**Comment by trajano** [comment]
You may want to expand it so people who may start to develop for https://github.com/microsoft/winget-cli can be aware and use this shim as well.  (Just recently discovered it via https://github.com/microsoft/PowerToys, its in beta but I am tempted to switch over eventually)

**Comment by 71** [comment]
Any name ideas? I updated the README to be more general in f6e7b02b56fe8024c28507e993fa520c081e207d.

**Comment by God-damnit-all** [comment]
schism, perhaps? And the code could be updated to read .schism or .shim files.

**Comment by hasufell** [comment]
You have another user: https://gitlab.haskell.org/haskell/ghcup-hs

Great way to emulate symlinks without admin privileges.

**Comment by 71** [comment]
At this point the most significant "branding" of this project is the name of the repo (scoop-better-shimexe), and I'm not sure I want to change it since it would likely break some things. Other than that and the Scoop mention in the README, the project is quite generic IMO and I'm not sure we do need to remove that branding.

@hasufell I believe you're talking about [this issue](https://gitlab.haskell.org/haskell/ghcup-hs/-/issues/130) specifically? If so, would the name of the repo be a problem?

**Comment by hasufell** [comment]
> @hasufell I believe you're talking about this issue specifically? If so, would the name of the repo be a problem?

Correct. I don't think the name is a problem, but it surely took some time to find this repo. I also stumbled over the proprietary chocolatey shimgen.

**Comment by God-damnit-all** [comment]
Well, let me rephrase my initial post. I had actually come across this repo beforehand, sometime in 2019. I essentially passed it up because Scoop didn't look useful, at least not for me. I didn't realize it could be used standalone until around the time I came across it again and then filed this issue.
